actions/v2/admin/internal/mapping/user.go

```diff
@@ -1,24 +1,24 @@
 package mapping
 
 import (
-	"github.com/bitcoin-sv/spv-wallet/engine/utils"
 	"github.com/bitcoin-sv/spv-wallet/engine/v2/users/usersmodels"
 	"github.com/bitcoin-sv/spv-wallet/models/response/adminresponse"
+	"github.com/samber/lo"
 )
 
-// CreatedUserResponse maps a user to a response
-func CreatedUserResponse(u *usersmodels.User) adminresponse.User {
+// UserToResponse maps a user to a response
+func UserToResponse(u *usersmodels.User) adminresponse.User {
 	return adminresponse.User{
 		ID:        u.ID,
 		CreatedAt: u.CreatedAt,
 		UpdatedAt: u.UpdatedAt,
 		PublicKey: u.PublicKey,
-		Paymails:  utils.MapSlice(u.Paymails, UsersPaymailResponse),
+		Paymails:  lo.Map(u.Paymails, UsersPaymailToResponse),
 	}
 }
 
-// UsersPaymailResponse maps a user's paymail to a response
-func UsersPaymailResponse(p *usersmodels.Paymail) adminresponse.Paymail {
+// UsersPaymailToResponse maps a user's paymail to a response
+func UsersPaymailToResponse(p *usersmodels.Paymail, _ int) adminresponse.Paymail {
 	return adminresponse.Paymail{
```

Comment:
And this is something problematic, because we probably would like to share the same method for mapping paymail to response. 
@chris-4chain maybe you have some idea for that?